File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p02577/p02577_num1_parsed.py
N = int(input())

# Calculate the sum of the digits in N
digit_sum = 0
for digit in str(N):
    digit_sum += int(digit)

# Check if the sum of digits is a multiple of 9
if digit_sum % 9 == 0:
    print("Yes")
else:
    print("No")

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_cpp/p02577/p02577_num1_parsed.cpp
#include <iostream>
#include <string>

using namespace std;

bool isMultipleOfNine(string n) {
    int sum = 0;
    for (char c : n) {
        sum += c - '0';
    }
    return (sum % 9 == 0);
}

int main() {
    string n;
    cin >> n;

    if (isMultipleOfNine(n)) {
        cout << "Yes" << endl;
    } else {
        cout << "No" << endl;
    }

    return 0;
}